contribution.md

```diff
@@ -0,0 +1,56 @@
+# How can I contribute?
```

Comment:
This filename should be CONTRIBUTING.md. More details [here](https://docs.github.com/en/github/building-a-strong-community/setting-guidelines-for-repository-contributors)

Could you fix that please? (notice the upper case too)